packages/webapp/src/delegates/components/delegate-funds-available.tsx

```diff
@@ -0,0 +1,97 @@
+import { useState } from "react";
+import {
+    assetToString,
+    Button,
+    Heading,
+    onError,
+    sumAssetStrings,
+    Text,
+    useDistributionsForAccount,
+    useMemberByAccountName,
+    useUALAccount,
+} from "_app";
+
+import { withdrawDelegateAvailableFunds } from "../transactions";
+
+interface Props {
+    account: string;
+}
+
+export const DelegateFundsAvailable = ({ account }: Props) => {
+    const [ualAccount] = useUALAccount();
+    const { data: member } = useMemberByAccountName(account);
+    const { data: distributions } = useDistributionsForAccount(account);
+    const [isLoading, setIsLoading] = useState(false);
+
+    const availableFunds = distributions
+        ? sumAssetStrings(
+              distributions.map((distribution) => distribution.balance)
+          )
+        : undefined;
+
+    // omit component if it's not a current delegate and available funds are empty
+    if ((!member || !member.election_rank) && !availableFunds) {
+        return null;
+    }
+
+    const nextDisbursementDate = "October 15, 2021"; // TODO: how to calculate this? create fn.
```

Comment:
Let's add a TODO in the code at minimum just to track this as an open item

Reply:
I'd prefer that we either figure out how to calculate this _in this PR_ OR omit the "Check back on x..." sentence entirely and only expose it once it's accurate. I'm getting wary of merging hard-coded (and inaccurate) placeholder information this close to launching our next milestone.